build.gradle

```diff
@@ -399,20 +399,17 @@ spotless {
   }
 }
 
-task setupLocalRepo {
-  println 'Setting up default repo configs'
+task setupWorkspace {
+  println 'Setting up default git config'
   def gitConfig = [
-      'core.hooksPath' : 'gradle/githooks',
+      'core.hooksPath' : 'gradle/git/hooks',
+      'core.excludesFile' : 'gradle/git/ignore-paths',
+      'blame.ignoreRevsFile' : 'gradle/git/blame-ignore-revs',
```

Comment:
I have 3 comments here:
1. I think we should use a filename : `.git-blame-ignore-revs` so that GitHub UI also filter these out ([Reference](https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/))
2. For the `excludesFile`, I think using `.gitignore` is better since it is universally known and also already in-repo.
3. If we still choose to use a non-standard path, we should consider moving the `gradle/git` directory to root of the project to make it easier to find.

Reply:
Thanks, updated the PR.